app/utilities/schema.py

```diff
@@ -177,7 +178,44 @@ def load_schema_from_url(schema_url, language_code):
 
     constructed_schema_url = f"{schema_url}?language={language_code}"
 
-    req = requests.get(constructed_schema_url)
+    session = requests.Session()
+
+    retries = Retry(
+        total=3,
+        allowed_methods=frozenset({"GET"}),
+        backoff_factor=0.1,
+        status_forcelist=[
+            408,
+            429,
+            500,
+            501,
+            502,
+            503,
+            504,
+            505,
+            506,
+            507,
+            509,
+            510,
+            511,
+        ],
+    )  # Codes to retry according to Google Docs
+
+    session.mount("https://", HTTPAdapter(max_retries=retries))
```

Comment:
I think we should do this for both http and https. We should get https, but we shouldn't care since we are just fetching artifacts. We currently don't enforce that it must be https, but that is something we should do.

Reply:
Added `session.mount("http://", HTTPAdapter(max_retries=retries))`, thanks.